### DISCUSSION TITLE
How to package into an exe

### DISCUSSION THREAD

**Discussion by pmarbibi12** [Akascape/CTkTable#48] [Q&A] [answered]
so with customtkinter, you have to add the data for customtkinter for pyinstaller to properly package the script into an exe file. Do you also have to do the same for CTkTable?

for reference, you have to package your script like this for customtkinter:

pyinstaller --noconfirm --onedir --windowed --add-data "CustomTkinter Location/customtkinter;customtkinter/"  "Path to Python Script"

**Comment by Akascape** [comment, marked as answer]
@pmarbibi12 
> Do you also have to do the same for CTkTable?

Not necessary, pyinstaller will automatically pack it. But in case you find any errors, you can pack it separately.

A full guide for pyinstaller is given here: https://github.com/TomSchimansky/CustomTkinter/discussions/939

**Reply by pmarbibi12** [reply]
Thanks for the quick response! I tried it anyway and it's working fine! Great widget!
